edi_storage_oca/components/listener.py

```diff
@@ -14,21 +14,16 @@ class EdiStorageListener(Component):
     def _move_file(self, storage, from_dir_str, to_dir_str, filename):
         from_dir = PurePath(from_dir_str)
         to_dir = PurePath(to_dir_str)
-        if filename not in storage.list_files(from_dir.as_posix()):
-            # The file might have been moved after a previous error.
+        # TODO:
+        # - storage.list_files now includes path in fs_storage, breaking change
+        # - we remove path
```

Comment:
not sure to understand this comment

Reply:
In `fs_storage`, the `list_files` method now returns filenames  with path. Ex: folderA/folderB/filename,..
About `backend_storage`, it just returns the filenames. So we make some changes on codes

Reply:
so... is it still a TODO or not?